pkg/kubelet/eviction/helpers_test.go

```diff
@@ -3250,3 +3382,52 @@ func TestEvictonMessageWithResourceResize(t *testing.T) {
 		})
 	}
 }
+
+func TestStatsNotFoundForPod(t *testing.T) {
+	pod1 := newPod("fake-pod1", defaultPriority, []v1.Container{
+		newContainer("fake-container1", newResourceList("", "", ""), newResourceList("", "", "")),
+	}, nil)
+	pod2 := newPod("fake-pod2", defaultPriority, []v1.Container{
+		newContainer("fake-container2", newResourceList("", "", ""), newResourceList("", "", "")),
+	}, nil)
+	statsFn := func(pod *v1.Pod) (statsapi.PodStats, bool) {
+		return statsapi.PodStats{}, false
+	}
+
+	for _, test := range []struct {
+		description string
+		compFunc    func(stats statsFunc) cmpFunc
+	}{
+		{
+			description: "process",
+			compFunc:    process,
+		},
+		{
+			description: "memory",
+			compFunc:    memory,
+		},
+		{
+			description: "exceedMemoryRequests",
+			compFunc:    exceedMemoryRequests,
+		},
+		{
+			description: "exceedDiskRequests",
+			compFunc: func(stats statsFunc) cmpFunc {
+				return exceedDiskRequests(stats, nil, "")
+			},
+		},
+		{
+			description: "disk",
+			compFunc: func(stats statsFunc) cmpFunc {
+				return disk(stats, nil, "")
+			},
+		},
+	} {
+		t.Run(test.description, func(t *testing.T) {
+			result := test.compFunc(statsFn)(pod1, pod2)
+			if result != 0 {
+				t.Errorf("expected 0 but got %d", result)
+			}
```

Comment:
The same here, please use [testify](https://github.com/stretchr/testify) assert/require APIs.